fish@kriegcc/README.md

```diff
@@ -198,10 +198,20 @@ Thank you to all contributors and the open-source community for their support an
 
 [GPL-3.0-or-later](https://spdx.org/licenses/GPL-3.0-or-later.html)
 
-See: [License](./COPYING)
+See: [License][license]
 
 ## Resources
 
 ### Miscellaneous
 
 - Wanda The Fish fan page (by ePsiLoN, last update 2013): http://wandathefish.com/blog/wanda-fan-site/
+
+<!-- adjust URLs so they work on the Cinnamon Applet Spices website -->
```

Comment:
Is this okay? Or better store the images somewhere else?

Reply:
These look fine I think.